### DISCUSSION TITLE
Installation of O3DE in non empty folder should show warning

### DISCUSSION THREAD

**Discussion by N2KillU** [o3de/o3de#12931] [Ideas]
When selecting a new folder or disk the extra directory for O3DE will not be created. All the individual files contained in O3DE->(Engine Version) directories will be installed directly in the selected path, potentially cluttering a user directory. A warning that no additional directories will be created would be greatly appreciated.

**Comment by AMZN-daimini** [comment]
@o3de/sig-build
